application/helpers/expressions/em_manager_helper.php

```diff
@@ -9800,6 +9752,75 @@ public static function htmlSpecialCharsUserValue($string)
         $string = str_replace(["{","}"], ["&#123;","&#125;"], $string);
         return $string;
     }
+
+    /**
+     * Save the starting values to the database
+     */
+    public static function SaveStartingValues()
+    {
+        // Init
+        $LEM =& LimeExpressionManager::singleton();
+
+        // If not starting values, retun
+        if (!isset($_SESSION[$LEM->sessid]['startingValues'])) return false;
```

Comment:
Can you check this code with PSR-12, please?

Reply:
Done. Thanks!

Reply:
:partying_face: